src/dataServices/dynamoDbBundleService.ts

```diff
@@ -190,7 +193,11 @@ export class DynamoDbBundleService implements Bundle {
         const idItemsFailedToRead: string[] = [];
         for (let i = 0; i < itemResponses.length; i += 1) {
             const itemResponse = itemResponses[i];
-            if (itemResponse instanceof ResourceNotFoundError) {
+            // allow for update as create scenario
+            if (
+                itemResponse instanceof ResourceNotFoundError &&
+                !(itemsToLock[i].operation === 'update' && this.updateCreateSupported)
```

Comment:
nit: I would flip the `if` block to do the Boolean check first 
```suggestion
                this.updateCreateSupported && itemResponse instanceof ResourceNotFoundError &&
                !(itemsToLock[i].operation === 'update')
```

Reply:
The two expressions don't seem to evaluate the same. Now if `this.updateCreateSupported` is false the whole expression evaluates to false no matter what the operation is or if there's an error or not. Am I missing something?

Reply:
oh yep you are right there are parenthesis. Please use existing!